packages/wp-utils/src/bindToCallbacks.ts

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const bindToCallbacks = (callbacks, config) => {
+
+function bindToCallbacks(callbacks: Record<string, unknown>, config: object) {
```

Comment:
Here I think generics are needed for the config object. We need the callbacks to be typed so that they are functions taking a parameter of the type that the config object has, hence the generic type. Something like this should work:

```suggestion
function bindToCallbacks<T>(callbacks: Record<string, (config: T) => unknown>, config: T):Record<string, unknown> {
```

Additionally, if this is only used in places, where all the callbacks return an object of the same type, the `unknown` can be replaced by another generic, e.g. just `S`.

Reply:
Seems to work.